.gitignore

```diff
@@ -2,7 +2,6 @@ dist
 doc
 parsers/*.js
 node_modules
-.vscode/
```

Comment:
Hm I hadn't thought of that, because I never have any "personal settings" in this folder (I have them in my user's VSCode settings, not per-project).

But I guess it's possible that someone has project-specific settings in this folder... In that case I guess it should be their responsibility not to commit them, and there are git settings to ignore some files. But I can see how that is error prone and probably not the best thing to do. So I guess we could start by just allowing this one and see if we need anything else in the future.

Although before doing that, I'd like to hear some examples. Do you have an example of per-project settings you would have in this folder that cannot be in your user settings? It seems to me like most of those could be in the repo as well.